api.planx.uk/server.ts

```diff
@@ -477,6 +478,9 @@ app.get(
   graphQLVoyagerHandler({ graphQLURL: "/introspect-all", validateUser: true })
 );
 
+// use with query param `insert=true` (optional)
```

Comment:
nit: This is in the body, not a query param

Reply:
A type for the body might be helpful also as opposed to just a comment here. The main reason for this is that if somebody were to post the body of `{ insert: "false" }` this would currently incorrectly evaluate `shouldInsert` to  be `true`.

```ts
const shouldInsert = Boolean(req.body?.insert);
```

I think that we probably need to have a better approach to handling validation at the API level across the board - something to discuss!